rust/src/exchange/etsi/decentralized_environmental_notification_message.rs

```diff
@@ -15,14 +15,14 @@ use crate::exchange::etsi::decentralized_environmental_notification_message::Rel
 };
 use crate::exchange::etsi::reference_position::ReferencePosition;
 use crate::exchange::etsi::{
-    etsi_now, heading_from_etsi, speed_from_etsi, timestamp_from_etsi, PathHistory,
-    PositionConfidence,
+    etsi_now, heading_from_etsi, speed_from_etsi, PathHistory, PositionConfidence,
 };
 use crate::exchange::message::content::Content;
 use crate::exchange::message::content_error::ContentError;
 use crate::exchange::mortal::Mortal;
 use crate::mobility::mobile::Mobile;
 use crate::mobility::position::Position;
+
```

Comment:
Blank line line 25 in decentralized_environmental_notification_message.rs (try again creating comment from commit tab and line in code to see if the reference is added to the comment)

Reply:
ok